extensions/apidoc/helpers/ApiIndexer.php

```diff
@@ -25,18 +25,18 @@ class ApiIndexer extends Indexer
      */
     protected function generateFileInfo($file, $contents, $basePath, $baseUrl)
     {
-        // create file entry
-        if (preg_match('~<h1>(.*?)</h1>~s', $contents, $matches)) {
+		// create file entry
+        if (preg_match('~<h1>(.*|\r*|\n*?)</h1>~u', $contents, $matches)) {
```

Comment:
this has been handled by `/s` modifier before: http://php.net/manual/de/reference.pcre.pattern.modifiers.php
also adding `|` always needs new parenthesis: `(.*|\r*|\n*)`